test/integration/test/async-translation-summarization.test.js

```diff
@@ -0,0 +1,178 @@
+const fs = require('fs');
+const clientHelper = require('../src/client-helper');
+const { SummarizationJobStatus } = require('../../../dist/src/models/async/SummarizationJobStatus');
+const { TranslationJobStatus } = require('../../../dist/src/models/async/TranslationJobStatus');
+const { JobStatus } = require('../../../dist/src/models/JobStatus');
+const { NlpModel } = require('../../../dist/src/models/NlpModel');
+
+test('async translation/summarization submit local file', async () => {
+    const client = clientHelper.getAsyncClient();
+    const options = new Object();
+    options.metadata = 'Node sdk async translation/summarization submit local file';
+    options.delete_after_seconds = 50000;
+    options.language = "en";
+
+    options.summarization_config = {
+        type:'bullets',
+        model:'premium',
```

Comment:
if we are using the enum, why don't you use the enum here?
otherwise, why don't use a string literal e.g. `NlpModel = 'standard' | 'premium'`?